testsuite/.rubocop_todo.yml

```diff
@@ -358,14 +361,11 @@ Style/DateTime:
     - 'features/step_definitions/datepicker_steps.rb'
     - 'features/step_definitions/setup_steps.rb'
 
-# Offense count: 14
+# Offense count: 4
 # Cop supports --auto-correct.
 Style/DisableCopsWithinSourceCodeDirective:
```

Comment:
As I told to Chris, I would disable this Cop for all.

Reply:
I agree but  I would do this in another PR where we tackle the RuboCop issues one by one. See https://github.com/SUSE/spacewalk/issues/14192